.github/workflows/main.yml

```diff
@@ -125,9 +124,9 @@ jobs:
         nr: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
     steps:
     - uses: actions/checkout@v1
-    - name: download git-64-portable
+    - name: download git-sdk-64-minimal
       shell: bash
-      run: a=git-64-portable && mkdir -p $a && curl -# https://wingit.blob.core.windows.net/ci-artifacts/$a.tar.xz | tar -C $a -xJf -
+      run: a=git-sdk-64-minimal && mkdir -p $a && curl -# https://wingit.blob.core.windows.net/ci-artifacts/$a.tar.xz | tar -C $a -xJf -
```

Comment:
To avoid hammering our `ci-artifacts` container, in https://github.com/gitgitgadget/git/pull/632 I now also use `git-sdk-64-minimal`. While this does not test the intended workflow (when building with Visual Studio, we do _not_ want to require users to download Git for Windows' SDK just to run the tests, we want the tests to run in a regular Git Bash), it needs to be good enough for now.

Reply:
(In other words, I do not ask you to change anything here.)